app/features/send/send-store.ts

```diff
@@ -200,6 +224,7 @@ export const createSendStore = ({
   getInvoiceFromLud16,
   createCashuSendQuote,
   getCashuSendSwapQuote,
+  getSparkSendQuote,
```

Comment:
should we be consistent with naming? we have createCashuSendQuote but get for send swap and spark send